security-command-center/snippets/src/test/java/vtwo/MuteFindingIT.java

```diff
@@ -173,18 +181,20 @@ public void testUpdateMuteRules() throws IOException {
   }
 
   @Test
-  public void testMuteUnmuteFinding() throws IOException {
+  public void testSetMuteFinding() throws IOException {
     Finding finding = SetMuteFinding.setMute(FINDING_1.getName());
     assertThat(finding.getMute()).isEqualTo(Mute.MUTED);
     finding = SetUnmuteFinding.setUnmute(FINDING_1.getName());
     assertThat(finding.getMute()).isEqualTo(Mute.UNMUTED);
+    finding = SetMuteUndefinedFinding.setMuteUndefined(FINDING_1.getName());
```

Comment:
issue: don't check for a result; instead parse stdout to ensure that something was printed.

Reply:
I'd like to keep this code as-is, given that it's consistent with the instructions in this repo.